config/routes.rb

```diff
@@ -9,6 +9,8 @@
     resources :comments, only: [:create]
     resources :likes, only: [:create, :destroy]
   end
-
+  post '/add_friend', to: 'users#add_friend', as: 'add_friend'
+  post '/accept_friendship', to: 'users#accept_friendship', as: 'accept_friendship'
+  post '/refuse_friendship', to: 'users#refuse_friendship', as: 'refuse_friendship'
```

Comment:
After reading the documentation about the RESTful patter, I still don't understand your request, could you please explain it to me in a more explicit way? If you could give me an example it would be great also! Thanks